ipa-core/src/bin/report_collector.rs

```diff
@@ -430,6 +431,9 @@ async fn hybrid(
     count: usize,
     set_fixed_polling_ms: Option<u64>,
 ) -> Result<(), Box<dyn Error>> {
+    // twice the size of TCP MSS. This may get messed up if TCP options are used which is not
+    // in our control, but hopefully fragmentation is not too bad
+    const BUF_SIZE: NonZeroUsize = NonZeroUsize::new(1072).unwrap();
```

Comment:
@andyleiserson do you have any suggestion on picking up the buffer size?

Reply:
I think what's important is that the kernel has at least a full packet's worth of data, so that it can send full packets.